src/components/todoItem/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'classnames';
+
+interface Props {
+  title: string;
+  id: number;
+  completed: boolean;
+  onDelete: (id: number) => void;
+  onIsComplitedUpdate: (
+    id: number,
+    complitedCurrVal: boolean,
+  ) => void;
```

Comment:
There is a typo in the 'complitedCurrVal' parameter name. It should be 'completedCurrVal'.